boards/Makefile.common

```diff
@@ -127,8 +134,14 @@ SIZE      ?= $(TOOLCHAIN)-size
 OBJCOPY   ?= $(TOOLCHAIN)-objcopy
 OBJDUMP   ?= $(TOOLCHAIN)-objdump
 
+# Set default flags to produce binary from .elf
+OBJCOPY_FLAGS ?= --output-target=binary
+# Add default flags to cargo. Boards can add additional options
+CARGO_FLAGS += $(VERBOSE) --target=$(TARGET) --package $(PLATFORM)  --target-dir=$(TARGET_DIRECTORY)
 # Set the default flags we need for objdump to get a .lst file.
-OBJDUMP_FLAGS += --disassemble-all --source --section-headers
+OBJDUMP_FLAGS ?= --disassemble-all --source --section-headers --demangle
```

Comment:
Since we use LLVM tools, does that mean that `--demangle` does something wrong? Or it just doesn't do anything? It is ok to have it here by default?

Reply:
it partially do the right thing, partially do nothing. may be this is because it's C++ focused. gcc's objdump do it much better and also provide comments on addresses of data symbols loaded [this probably is from relocation records], as this is much more informative than just load constant 0x118c0 in register.